src/modules/rlm_smtp/all.mk

```diff
@@ -1,12 +1,22 @@
 #  Check to see if we libfreeradius-curl, as that's a hard dependency
 #  which in turn depends on json-c.
 TARGETNAME	:=
+VERSION		:=
 -include $(top_builddir)/src/lib/curl/all.mk
 TARGET		:=
 
 ifneq "$(TARGETNAME)" ""
+
+#  Require cURL >= 7.56.0
+CURL_X		:= $(word 1,$(subst ., ,$(VERSION)))
+CURL_Y		:= $(word 2,$(subst ., ,$(VERSION)))
+CURL_GOOD	:= $(shell test $(CURL_X) -gt 7 -o \( $(CURL_X) -eq 7 -a $(CURL_Y) -ge 56 \) && echo true)
```

Comment:
The issue with this is it forks for every invocation of make, no matter what the target, it'd be better if you could do it with builtins.